content/en/architectures/adobe/index.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Scaling Adobe’s Service Delivery Foundation with a Cell-based Architecture
+date: 2024-10-11
+org_name: Adobe
+org_team: Developer Platforms
+org_url: https://www.adobe.com/
+org_logo_filename: ./images/Adobe_Wordmark_RGB_Red.svg
```

Comment:
@cjyabraham do you happen to know why the logo is not rendering? Is it only supported if the logo is in the root of the org dir instead of in a subfolder like this?

Reply:
Try just `images/Adobe_Wordmark_RGB_Red.svg`

Reply:
@cjyabraham This worked. Thanks!